Submitted By:            Randy McMurchy <randy_at_linuxfromscratch_dot_org>
Date:                    2005-10-04
Initial Package Version: 1.0.9
Upstream Status:         N/A (package is not actively maintained)
Origin:                  Randy McMurchy
Description:             Fixes GCC-4.0.x build problems

$LastChangedBy$
$Date$


--- GConf-1.0.9-orig/backends/bdb.c	2001-10-17 03:43:01.000000000 +0000
+++ GConf-1.0.9/backends/bdb.c	2005-10-04 02:19:11.000000000 +0000
@@ -331,7 +331,7 @@
 
 /* } */
 
-static guint32
+guint32
 get_dir_id (BDB_Store * bdb, const char *dir)
 {
   int ret;
@@ -405,7 +405,7 @@
 
 /* { Functions to create directories and maintain the dir hierarchy */
 
-static guint32
+guint32
 get_or_create_dir (BDB_Store * bdb, const char *dir)
 {
   guint32 parent_id;
